### DISCUSSION TITLE
posts/back-again-2023/

### DISCUSSION THREAD

**Discussion by giscus** [jerrylususu/jerrylususu.github.io#28] [Blog Comments]
# posts/back-again-2023/

又是很久没更新博客了，看了看记录上次更新已经差不多是一年前了。和上次一样，这一年中依然发生了很多：完成了硕士学业，解封前跨过重重关卡回国，入

https://nekonull.me/posts/back-again-2023/

**Comment by note4** [comment]
访问响应速度有点慢，要不要考虑下  gitlab onrender vercel

**Reply by jerrylususu** [reply]
感谢回复！目前更新频率不高，还没有什么折腾的打算；之前已经有一个备案的国内域名，后续会建立国内镜像彻底解决速度问题。
